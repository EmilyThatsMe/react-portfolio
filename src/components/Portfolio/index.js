import React, { useState } from 'react';
import kanye from '../../assets/images/kanye.jpg';
import dog from '../../assets/images/dog.jpg';
import profile from '../../assets/images/profile.jpg';
import run from '../../assets/images/run.jpg';
import quiz from '../../assets/images/quiz.jpg';
import notes from '../../assets/images/notes.jpg';

function Portfolio() {
  return (
    <section className="portfolio" id="portfolio">
      <h1>Portfolio</h1>
      <div className="flex-row">
        <div className="img--container">
          <img src={kanye} className="img-thumbnail mx-1"></img>
          <span>
            <a href="https://concord511.github.io/Quote-Battle/">
              Kanye Guess - HTML, CSS, Javascript
            </a>
            <br></br>
            <a href="https://github.com/Concord511/Quote-Battle">Github</a>
          </span>
        </div>
        <div className="img--container">
          <img src={dog} className="img-thumbnail mx-1"></img>
          <span>
            <a href="https://secret-sea-39551.herokuapp.com/">
              Rate My Dog - SQL, Sequelize
            </a>
            <br></br>
            <a href="https://github.com/cassie-s/the-dog-blog">Github</a>
          </span>
        </div>
        <div className="img--container">
          <img src={profile} className="img-thumbnail mx-1"></img>
          <span>
            <a href="https://github.com/EmilyThatsMe/profile-generator">
              Profile generator - Node, Express, Inquirer
            </a>
            <br></br>
            <a href="https://github.com/EmilyThatsMe/profile-generator">
              Github
            </a>
          </span>
        </div>
        <div className="img--container">
          <img src={quiz} className="img-thumbnail mx-1"></img>
          <span>
            <a href="https://emilythatsme.github.io/code-quiz/">
              Code Quiz - CSS, Javascript, Moment.js
            </a>
            <br></br>
            <a href="https://github.com/EmilyThatsMe/code-quiz">Github</a>
          </span>
        </div>
        <div className="img--container">
          <img src={run} className="img-thumbnail mx-1"></img>
          <span>
            <a href="https://emilythatsme.github.io/run-buddy/">
              Run Buddy - HTML, CSS
            </a>
            <br></br>
            <a href="https://github.com/EmilyThatsMe/run-buddy">Github</a>
          </span>
        </div>
        <div className="img--container">
          <img src={notes} className="img-thumbnail mx-1"></img>
          <span>
            <a href="https://emilythatsme.github.io/express-note-taker/">
              Note Taker - Node, Express
            </a>
            <br></br>
            <a href="https://github.com/EmilyThatsMe/express-note-taker">
              Github
            </a>
          </span>
        </div>
      </div>
    </section>
  );
}
export default Portfolio;
