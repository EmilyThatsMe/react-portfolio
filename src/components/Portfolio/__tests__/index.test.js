import React from 'react';
import { render, cleanup } from '@testing-library/react';
import '@testing-library/jest-dom/extend-expect';
import Portfolio from '..';

afterEach(cleanup);

describe('Portfolio component', () => {
  // First test
  it('renders', () => {
    render(<Portfolio />);
  });

  // Second test
  it('matches snapshot DOM node structure', () => {
    const { asFragment } = render(<Portfolio />);
    expect(asFragment()).toMatchSnapshot();
  });
});
