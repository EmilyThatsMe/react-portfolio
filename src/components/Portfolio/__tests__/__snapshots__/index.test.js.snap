// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Portfolio component matches snapshot DOM node structure 1`] = `
<DocumentFragment>
  <section
    class="portfolio"
    id="portfolio"
  >
    <h1>
      Portfolio
    </h1>
    <div
      class="flex-row"
    >
      <div
        class="img--container"
      >
        <img
          class="img-thumbnail mx-1"
          src="kanye.jpg"
        />
        <span>
          <a
            href="https://concord511.github.io/Quote-Battle/"
          >
            Kanye Guess - HTML, CSS, Javascript
          </a>
          <br />
          <a
            href="https://github.com/Concord511/Quote-Battle"
          >
            Github
          </a>
        </span>
      </div>
      <div
        class="img--container"
      >
        <img
          class="img-thumbnail mx-1"
          src="dog.jpg"
        />
        <span>
          <a
            href="https://secret-sea-39551.herokuapp.com/"
          >
            Rate My Dog - SQL, Sequelize
          </a>
          <br />
          <a
            href="https://github.com/cassie-s/the-dog-blog"
          >
            Github
          </a>
        </span>
      </div>
      <div
        class="img--container"
      >
        <img
          class="img-thumbnail mx-1"
          src="profile.jpg"
        />
        <span>
          <a
            href="https://github.com/EmilyThatsMe/profile-generator"
          >
            Profile generator - Node, Express, Inquirer
          </a>
          <br />
          <a
            href="https://github.com/EmilyThatsMe/profile-generator"
          >
            Github
          </a>
        </span>
      </div>
      <div
        class="img--container"
      >
        <img
          class="img-thumbnail mx-1"
          src="quiz.jpg"
        />
        <span>
          <a
            href="https://emilythatsme.github.io/code-quiz/"
          >
            Code Quiz - CSS, Javascript, Moment.js
          </a>
          <br />
          <a
            href="https://github.com/EmilyThatsMe/code-quiz"
          >
            Github
          </a>
        </span>
      </div>
      <div
        class="img--container"
      >
        <img
          class="img-thumbnail mx-1"
          src="run.jpg"
        />
        <span>
          <a
            href="https://emilythatsme.github.io/run-buddy/"
          >
            Run Buddy - HTML, CSS
          </a>
          <br />
          <a
            href="https://github.com/EmilyThatsMe/run-buddy"
          >
            Github
          </a>
        </span>
      </div>
      <div
        class="img--container"
      >
        <img
          class="img-thumbnail mx-1"
          src="notes.jpg"
        />
        <span>
          <a
            href="https://emilythatsme.github.io/express-note-taker/"
          >
            Note Taker - Node, Express
          </a>
          <br />
          <a
            href="https://github.com/EmilyThatsMe/express-note-taker"
          >
            Github
          </a>
        </span>
      </div>
    </div>
  </section>
</DocumentFragment>
`;
